### ISSUE TITLE
[Bug]: can't find virsh domain from cgroup

### ISSUE THREAD

**Issue by Forza-tng** [netdata/netdata#13865]
### Bug description

 `cgroup-network-helper.sh` fails to get the correct domain/name for my virtual machines.

```
# /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/sys/fs/cgroup/machine/qemu-1-alpinelinux3.16.libvirt-qemu'

error: failed to get domain 'qemu-1-alpinelinux3.16'
```

```
 # virsh list
 Id   Name              State
---------------------------------
 1    alpinelinux3.16   running 
```

The cgroup is actually `/sys/fs/cgroup/machine/qemu-1-alpinelinux3.16.libvirt-qemu` but the shell script doesn't seem to handle the `qemu-1-` prefix. Removing it on the command line gives correct output:
```
# /opt/netdata/usr/libexec/netdata/plugins.d/cgroup-network-helper.sh --cgroup '/sys/fs/cgroup/machine/alpinelinux3.16.libvirt-qemu'
2022-10-22 22:29:12: cgroup-network-helper.sh: INFO: searching for network interfaces of cgroup '/sys/fs/cgroup/machine/alpinelinux3.16.libvirt-qemu'
vnet0 vnet0_lan
vnet1 vnet1_inet
``` 

### Expected behavior

Correct virsh domain should be detected. 

### Steps to reproduce

1.
2.
3.
...

### Installation method

from git

### System info

```shell
Linux 6.0.2-gentoo #1 SMP Thu Oct 20 12:21:52 CEST 2022 x86_64 AMD Athlon 3000G with Radeon Vega Graphics AuthenticAMD GNU/Linux
/etc/gentoo-release:Gentoo Base System release 2.8
/etc/lsb-release:DISTRIB_ID="Gentoo"
/etc/os-release:NAME=Gentoo
/etc/os-release:ID=gentoo
/etc/os-release:PRETTY_NAME="Gentoo Linux"
/etc/os-release:ANSI_COLOR="1;32"
/etc/os-release:VERSION_ID="2.8"
```

### Netdata build info

```shell
Version: netdata v1.36.0-248-nightly
Configure options:  '--prefix=/opt/netdata/usr' '--sysconfdir=/opt/netdata/etc' '--localstatedir=/opt/netdata/var' '--libexecdir=/opt/netdata/usr/libexec' '--libdir=/opt/netdata/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--enable-lto' '--enable-ml' '--with-bundled-protobuf' 'CFLAGS=-O3 -march=native -pipe' 'LDFLAGS=-Wl,--as-needed -Wl,-O1 -Wl,--sort-common' 'CPPFLAGS=-O3 -march=native -pipe' 'CXXFLAGS=-O3 -march=native -pipe'
Install type: custom
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (bundled)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  YES
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    YES
    IPMI:                    NO
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

### Additional info

Originally asked in Discord: https://discord.com/channels/847502280503590932/1033477219592454266

**Comment by Ferroin** [comment]
Confirmed locally with a static install of v1.36.1.

**Comment by ilyam8** [comment]
> doesn't seem to handle the qemu-1- prefix

Thanks for the report and for finding the problem, @Forza-tng. Fixed in #13866.

**Comment by Forza-tng** [comment]
> > doesn't seem to handle the qemu-1- prefix
> 
> Thanks for the report and for finding the problem, @Forza-tng. Fixed in #13866.

That was quick!

Tested it and it looks like it works! 🐦
![Screenshot_20221023_004533](https://user-images.githubusercontent.com/68693597/197365033-6fbab40c-6604-4e89-8447-4fabcd62271c.png)
